config/database.yml

```diff
@@ -25,7 +25,7 @@ development:
   <<: *default
   database: rail_blog_app_development
   user: postgres
-  password: Elias2001
+  password: 52556
```

Comment:
- [ ] [ OPTIONAL] Never use the password directly, always use the environment variable to represent it & mention that environment file in the `.gitignore` file. [Read here ](http://railsapps.github.io/rails-environment-variables.html)how to implement  it.